tests/cms/models/chat/test_user_chat.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from django.conf import settings
+
+from integreat_cms.cms.models import UserChat
+
+
+@pytest.mark.django_db
+def test_update_mt_budget() -> None:
+    """
+    Test if words are being correctly added to the used MT budget and chat word count
+    """
+    chat = UserChat.objects.get(pk=1)
+    mt_budget_before = chat.region.mt_budget_used
+    total_words_generated_before = chat.total_words_generated
+    words_generated = 155
```

Comment:
```suggestion
    WORDS_GENERATED = 155
```
Nit: I would put this at the top and make it a constant :)